### DISCUSSION TITLE
如何解决超时问题

### DISCUSSION THREAD

**Discussion by iPodtouch6557** [Meloong-Git/PCL#3935] [提问] [answered]
<img width="583" alt="image" src="https://github.com/Hex-Dragon/PCL2/assets/118656946/7de3a203-c36c-46be-a91a-8160029e4bd6">
<img width="400" alt="image" src="https://github.com/Hex-Dragon/PCL2/assets/118656946/3b868128-cdee-4a71-b02b-6de07da35832">
所以真的只能挂梯子么

**Comment by MoYuan-CN** [comment]
Release/Snapshot 還是 Debug

**Reply by iPodtouch6557** [reply]
snapshot
别误会我老粉丝
换了个头像

**Reply by MoYuan-CN** [reply]
我看 itdog 全區都是正常的，不知道怎麽回事。
那應該就只能嘗試科學上網了

**Reply by iPodtouch6557** [reply]
运营商和地区的综合影响下，就是响应慢但是可以用

**Comment by LTCatt** [comment, marked as answer]
挂梯子，没救
